root/etc/e-smith/events/actions/nethserver-ejabberd-update-group

```diff
@@ -28,7 +28,7 @@ my $event = shift || die "Event name arg missing\n";
 my $groupName = shift;
 my $errors = 0;
 
-if ($groupName ne 'jabberadmins' ) {
+if ($groupName !~ 'jabberadmins' ) {
```

Comment:
Is it `!~ '^jabberadmins@'`?

Reply:
following if you modify a group in the panel user or group panel, the event call is not the same
- in user panel 
`Event: group-modify jabberadmins stephane toto`

- in group panel
`Event: group-modify jabberadmins@de-labrusse.fr helene stephane toto`

yes we could start by a `^` but the `@` is not welcome